### ISSUE TITLE
How to determine whether running workflows with act?

### ISSUE THREAD

**Issue by theowenyoung** [nektos/act#422]
Hi,

I can check if a workflow running in Github Actions by `${{ env.GITHUB_ACTIONS=='true'}}`, but act also define env `GITHUB_ACTIONS: true`, so how to determine whether running workflows with act?

There are some steps that I only want to run with Github, not locally.

Example:

```yaml
name: Build Site
on:
  workflow_dispatch:
  push:
jobs:
  run:
    runs-on: ubuntu-latest
    name: Run
    steps:
      - name: Create Pull Request
        uses: peter-evans/create-pull-request@v3
        if: env.GITHUB_ACTION=='true'
        with:
          token: ${{ secrets.PERSONAL_TOKEN }}
```

How to define `if` condition?

**Comment by austinsasko** [comment]
I think this is something that can have an env var for act specifically. e.g. in the runner context code I can add a line for 
`	env["ACT_LOCAL"] = "true"
` and some other code where needed (e.g. dist index.js in repo)

Then in the job you can specify 
`if: env.ACT_LOCAL=='true'` and have that job only execute when running from `act` on your local machine

Would this provide you the functionality you are looking for?

**Comment by theowenyoung** [comment]
@austinsasko yes, that totally solves my problem..

**Comment by torbjornvatn** [comment]
This is actually solved in #417 which was merged to master yesterday.
It introduces an `ACT=true` environment variable

**Comment by MattKiazyk** [comment]
@torbjornvatn when can we expect a release that has #417 in it?
